crates/bitwarden-auth/src/send_access/mod.rs

```diff
@@ -1,2 +1,13 @@
+mod access_token_request;
+mod access_token_response;
 mod client;
+
+/// Submodule containing the Send Access API request and response types.
+pub mod api;
```

Comment:
Rather than documenting the public modules here, I'd recommend instead going to the `api.rs` file and adding the comment like this to the first line:

```rust
//! Submodule containing the Send Access API request and response types.
```

This will have the same effect, but I think it makes more sense to have the comment with the file itself.

Reply:
03b64e84f07921f98a689f71c7acd51168fc211e